fetch-spl.sh

```diff
@@ -40,6 +40,7 @@ fetch_program() {
 
 fetch_program token 3.1.0 TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA BPFLoader2111111111111111111111111111111111
 fetch_program memo  1.0.0 Memo1UhkJRfHyvLMcVucJwxXeuD728EqVDDwQDxFMNo BPFLoader1111111111111111111111111111111111
```

Comment:
```suggestion
```

Reply:
backwards incompatible!

Reply:
I don't mind tossing it, just didn't think it would do any harm.

Reply:
actually yeah that was a bad idea.  But `program-test/src/programs.rs` should then include both memo programs too right?

Reply:
Yeah, that was my laziness. I'll add v1 / v2 modules

Reply:
To be clear, this memo is v2 despite the 3.0.0 semver right?

Reply:
It's the second memo program we've published to mainnet, yes.  Using spl_memo_1_0 and spl_memo_3_0 is probably the least confusing from a Cargo.toml/program.rs perspective

Reply:
sounds good, thanks